.github/workflows/release-build.yml

```diff
@@ -87,7 +86,8 @@ jobs:
     - uses: actions/checkout@v3
     - uses: ./.github/actions/just-setup
       with:
-        cache: false
+        index-cache: false
+        build-cache: false
```

Comment:
@passcod Setting `index-cache` and `build-cache` to `false` here somehow doesn't work, the CI still fetches the cache none the less.

Reply:
Though fetching the cache only adds a few seconds to the build and is probably ok.